isis/scripts/findfeaturesSegment

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import re 
+
+import kalasiris as kisis
+
+import subprocess
+import pvl
+
+from glob import glob
+import logging
+from pathlib import Path
+import logging as log
+import itertools 
+from copy import deepcopy
+import tempfile
+
+import plio
+from plio.io import io_controlnetwork as cnet
+from plio.io import isis_serial_number as sn
+
+MAX_LEN = 30000
+
+def parse_parameters():
+  # Use a regular expression to extract the parameter name and value from each
+  # key-value pair in the parameter string.
+  pattern = r'(\w+)=(\S+)'
+  
+  # Create a dictionary to store the parameter names and values.
+  params = {}
+  for arg in sys.argv[1:]:
+    # Extract the parameter name and value from the argument.
+    matches = re.findall(pattern, arg)
+    for match in matches:
+      name = match[0]
```

Comment:
kinda ghetto for now, but if we can expose ISIS's UI objects into python, we can write this in a way that is indistinguishable from a C++ ISIS app using Python (I think)